libkbfs/block_journal.go

```diff
@@ -783,22 +822,26 @@ func (j *blockJournal) getNextEntriesToFlush(
 		case addRefOp:
 			id, bctx, err := entry.getSingleContext()
 			if err != nil {
-				return blockEntriesToFlush{}, err
+				return blockEntriesToFlush{}, MetadataRevisionUninitialized, err
 			}
 
 			entries.adds.addNewBlock(
 				BlockPointer{ID: id, BlockContext: bctx},
 				nil, /* only used by folderBranchOps */
 				ReadyBlockData{}, nil)
 
+		case mdRevMarkerOp:
+			maxMDRevToFlush = entry.Revision
```

Comment:
This number can never go down, right? Maybe we should return an error if we see that it does.